### PR TITLE
Desing modal

### PR DESCRIPTION
**Pull request by ela-ocana-jalafund**
[AB#375](https://dev.azure.com/funjaladev/7867a1c5-5ac7-4c7f-baff-87d740bf4fc1/_workitems/edit/375)
https://dev.azure.com/funjaladev/DevLevel3/_backlogs/backlog/DevLevel3%20Team/Stories/?workitem=375

- Add an endpoint to get projects by skill name.
- Design a wireframe to show projects.
- Create a modal component to show projects.
- Integrate with the API

imagen

### CODE REVIEW

File: src/components/projects/ProjectsAPI.js [hunk 1]
```diff
@@ -60,3 +60,24 @@ export async function postProject(project) {
...
+
+export async function getProjectBySkills(skill) {
+  try {
+    const { data } = await axios.get(
+      `${API_HOST}/api/${API_VERSION}/${API_NAME}/${skill}`
+    );
+
+    return data;
+  } catch (error) {
+    // TODO error handling
```
Comment (jorge-lema-jalafund) [inline src/components/projects/ProjectsAPI.js]: remove comment

File: src/components/projects/ModalProjects.jsx [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    maxWidth: 400,
+    width: 400,
+  },
+}));
+
+export default function ProjectsModal({ searchProject, setModalProjects }) {
+  const classes = useStyles();
+  useEffect(() => {
+    async function data() {
+      // const projects = await getProjectsBySkill(searchProject);
```
Comment (jorge-lema-jalafund) [inline src/components/projects/ModalProjects.jsx]: what is this comment for?

**Review by jorge-lema-jalafund — CHANGES_REQUESTED** [review]

File: src/components/projects/ModalProjects.jsx (hunk 2 above)
Comment (ela-ocana-jalafund) [reply src/components/projects/ModalProjects.jsx:32]: At the moment I dont have endpoint

**Review by rodrigo-baldiviezo-jalafund — APPROVED** [review]

**Review by pedro-lima-jalafund — APPROVED** [review]
good job

**Review by jorge-lema-jalafund — APPROVED** [review]
approved

**Pull request merged**